crowdin/config.yml

```diff
@@ -1,15 +1,15 @@
 #
 # Your crowdin's credentials
 #
-"api_key_env": CROWDIN_API_KEY
-"project_identifier_env": CROWDIN_PROJECT_ID
+"api_token_env": CROWDIN_API_KEY
+"project_id_env": CROWDIN_PROJECT_ID
 "base_path_env": CROWDIN_BASE_PATH
 
 #
 # Choose file structure in crowdin
 # e.g. true or false
 #
-"preserve_hierarchy": false
```

Comment:
Has this changed meaning in v3 or is this an improvement to our config?

Reply:
This is a change in the V3. It is not possible anymore to use `dest` and `preserve_hierarchy` set to `true`